k2/csrc/cuda/debug.cuh

```diff
@@ -0,0 +1,169 @@
+// k2/csrc/cuda/errors.h
+
+// Copyright (c) 2020, Xiaomi Corporation ( authors: Meixu Song )
+
+// See LICENSE for clarification regarding multiple authors
+
+#ifndef K2_CSRC_CUDA_ERRORS_H_
```

Comment:
Please run the cpplint script in the `scripts` folder before committing.
It can help you spot the incorrect header guard.

Reply:
OK. I change it manually this time. And will find a way to make it take effect for my IDE.